GoogleUtilities/Example/Tests/Environment/GULHeartbeatDateStorageTest.m

```diff
@@ -18,10 +18,11 @@
 #import "GoogleUtilities/Environment/Private/GULHeartbeatDateStorage.h"
 
 @interface GULHeartbeatDateStorageTest : XCTestCase
-@property(nonatomic) NSURL *fileURL;
 @property(nonatomic) GULHeartbeatDateStorage *storage;
 @end
 
+NSString *const GULStorageHeartbeatTestFileName = @"GULStorageHeartbeatTest";
```

Comment:
Please add  prefix `k`. Also if you convert it to `static`,  then the name may be short e.g. 
`static NSString *const kTestFileName`

Reply:
i like that better. thanks!